### PR TITLE
DYN-6234 Use github issue forms

### PR DESCRIPTION
**Pull request by avidit**
### Purpose

Use github issue forms for filing issue.

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB 

### Release Notes

(FILL ME IN) Brief description of the fix / enhancement. **Mandatory section** 

### Reviewers

(FILL ME IN) Reviewer 1  (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Dynamo Revit Issues
```
Comment (QilongTang) [inline .github/ISSUE_TEMPLATE/config.yml:3]: Remove `Revit` here please

Comment (QilongTang) [reply .github/ISSUE_TEMPLATE/config.yml:3]: My mistake, looks like these are hyper links

**Comment by QilongTang** [comment]
Thank you @avidit These make sense, there is one test case though. For users without CER installed, their Dynamo session crash will guide them to create an issue here, Would that workflow also supported other than the manul trigger?

**Comment by avidit** [comment]
> Thank you @avidit These make sense, there is one test case though. For users without CER installed, their Dynamo session crash will guide them to create an issue here, Would that workflow also supported other than the manul trigger?

Yes. We can specify template in the url params. I will create a follow up PR for that.

**Review by QilongTang — APPROVED** [review]

**Pull request merged**